source/unified-test-format/unified-test-format.md

```diff
@@ -3522,19 +3536,6 @@ would need to represent streams as entities and support IO operations to directl
 entity. This may not be worth the added complexity if the existing operations provide adequate test coverage for GridFS
 implementations.
 
-### Support Client-side Encryption integration tests
-
-Supporting client-side encryption spec tests will require the following changes to the test format:
-
-- `json_schema` will need to be specified when creating a collection, via either the collection entity definition or
-    [initialData](#initialData).
-- `key_vault_data` can be expressed via [initialData](#initialData)
-- `autoEncryptOpts` will need to be specified when defining a client entity. Preparation of this field may require
-    reading AWS credentials from environment variables.
```

Comment:
It looks like the addition of `autoEncryptOpts` was the only change introduced in this PR.

Were the other changes not required? If so, why not?